src/preprocessing/passes/ackermann.cpp

```diff
@@ -21,7 +21,7 @@
 ￼**   https://cs.nyu.edu/media/publications/hadarean_liana.pdf
  **/
 
-#include "preprocessing/passes/bv_ackermann.h"
+#include "preprocessing/passes/ackermann.h"
 
 #include "options/bv_options.h"
```

Comment:
`#include "options/bv_options.h"` can be removed because it looks like it was only used for the `Assert` that this commit removes. However, `#include "theory/bv/theory_bv_utils.h"` is still needed for `bv::utils::mkAnd()`, which we should replace by a global `mkAnd` at some point but we don't have a good solution for that yet.

Reply:
Well, `mkAnd()` is pretty simple:
https://github.com/CVC4/CVC4/blob/923abd7000a2ab6e3c0776c59d159bdc3a4d9a52/src/theory/bv/theory_bv_utils.cpp#L324

I guess we could inline its code here and that way get rid of this dependency.